src/DynamoDbServiceProvider.php

```diff
@@ -40,11 +40,18 @@ protected function bindForApp($marshalerOptions = [])
             $config = [
                 'credentials' => [
                     'key' => config('services.dynamodb.key'),
-                    'secret' => config('services.dynamodb.secret'),
+                    'secret' => config('services.dynamodb.secret')
```

Comment:
Updated 😃  I've tried to be explicit with the fact the parameter is used for roles.